sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -24249,7 +24249,7 @@
               "allowCalculateOverride": false,
               "validate": {
                 "required": true,
-                "onlyAvailableItems": false,
```

Comment:
There is a radio on line 23928.

Reply:
That line should be connected to the change at line 23924.